process_request/routines.py

```diff
@@ -82,50 +83,53 @@ def is_submittable(self, item):
         """
         submit = True
         reason = None
-        if item.get("restrictions").lower() in ["closed"]:
+        if item["restrictions"] == "closed":
             submit = False
-            reason = "Item is restricted: {}".format(item.get("restrictions_text"))
-        elif item.get("preferred_format").lower() == "digital":
+            reason = "This object is currently unavailable for request. It will not be included in request. Reason: {}".format(item.get("restrictions_text"))
```

Comment:
While "item" was changed to "object" here, it's used in the following two "reason"s--it might make sense to use the same word for all 3 "reason"s.